<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarWindow</class>
 <widget class="QMainWindow" name="CalendarWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalendarWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="ModernCalendarWidget" name="calendarWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="actionswitchMode"/>
   <addaction name="actionopen_selected_noter"/>
   <addaction name="actioncreate_notes"/>
   <addaction name="actioncancel_mark"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionswitchMode">
   <property name="text">
    <string>SwitchMode</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionopen_selected_noter">
   <property name="text">
    <string>open_selected_noter</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actioncreate_notes">
   <property name="text">
    <string>create_notes</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actioncancel_mark">
   <property name="text">
    <string>cancel_mark</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModernCalendarWidget</class>
   <extends>QCalendarWidget</extends>
   <header>ModernCalendarWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
